src/main/java/com/atherys/towns/facade/PlotBorderFacade.java

```diff
@@ -0,0 +1,183 @@
+        if (plot.isPresent() && isPlayerViewingBorders(player)) {
+            BorderInfo borderInfo = new BorderInfo(blueWalls, player.getUniqueId(), plot.get().getNorthEastCorner(), plot.get().getSouthWestCorner());
+            addSelectionBorder(player, borderInfo);
+        }
+    }
+
+    public void showNewPlotSelectionBorders(Player player, Location<World> location) {
+        PlotSelection selection = plotSelectionFacade.getCurrentPlotSelection(player);
+        if (selection.isComplete() && isPlayerViewingBorders(player)) {
+            ParticleEffect effect = plotSelectionFacade.validatePlotSelection(selection, player, false, location) ? greenWalls : yellowWalls;
+            Plot plot = plotService.createPlotFromSelection(selection);
+            BorderInfo borderInfo = new BorderInfo(effect, player.getUniqueId(), plot.getNorthEastCorner(), plot.getSouthWestCorner());
+            addSelectionBorder(player, borderInfo);
+        }
+    }
+
+    public void initBorderTask() {
+        Task.builder().execute(task -> activeBorders.forEach((uuid, borderInfoSet) -> {
+            Sponge.getServer().getPlayer(uuid).ifPresent(player -> {
+                borderInfoSet.forEach(borderInfo -> {
+                    int xLength = MathUtils.getXLength(borderInfo.getNECorner(), borderInfo.getSWCorner());
+                    int zLength = MathUtils.getZLength(borderInfo.getNECorner(), borderInfo.getSWCorner());
+                    Vector3d particleLocationNE = new Vector3d(borderInfo.getNECorner().getX(), player.getPosition().getFloorY(), borderInfo.getNECorner().getY());
+                    Vector3d particleLocationSW = new Vector3d(borderInfo.getSWCorner().getX(), player.getPosition().getFloorY(), borderInfo.getSWCorner().getY());
+
+                    player.spawnParticles(borderInfo.getEffect(), particleLocationNE.add(1, 0, 0));
+                    player.spawnParticles(borderInfo.getEffect(), particleLocationSW.add(0, 0, 1));
+                    for (int i = 0; i <= zLength; i++) {
+                        player.spawnParticles(borderInfo.getEffect(), particleLocationNE.add(1, 0, i + 1));
+                        player.spawnParticles(borderInfo.getEffect(), particleLocationSW.sub(0, 0, i));
+                    }
+                    for (int i = 0; i <= xLength; i++) {
+                        player.spawnParticles(borderInfo.getEffect(), particleLocationNE.sub(i, 0, 0));
+                        player.spawnParticles(borderInfo.getEffect(), particleLocationSW.add(i + 1, 0, 1));
+                    }
+                });
+            });
+        })).intervalTicks(10).submit(AtherysTowns.getInstance());
+    }
+
+    public void setPlayerViewBorderStatus(Player player, boolean state) throws TownsCommandException {
+        Text.Builder statusText = Text.builder();
+        Text status = state ? Text.of(TextColors.GREEN, "Enabled") : Text.of(TextColors.RED, "Disabled");
```

Comment:
Side note: is there a localisation design throughout the plugins or is English the only option?